worker/util.go

```diff
@@ -100,3 +113,10 @@ func detectResources(conf config.Resources) config.Resources {
 // that doesn't do anything.
 func NoopTaskRunner(l TaskControl, c config.Worker, j *pbf.TaskWrapper) {
 }
+
+const (
+	b  = 1
+	kb = 1024 * b
+	mb = 1024 * kb
+	gb = 1024 * mb
```

Comment:
These are gibibytes (1024). Let's stick to SI units, which means these should be multiples of 1000. We might need to clarify this point in the TES docs as well.